resources/views/website/backend/layouts/chart.blade.php

```diff
@@ -0,0 +1,72 @@
+@extends('website.backend.layouts.main')
+
+@section('content')
+<canvas id="myChart" width="400" height="200"></canvas>
+<script src="{{ asset('bower_components/components-font-awesome/chart.js/dist/Chart.js') }}"></script>
+
+<script>
+    var updateChart = function () {
```

Comment:
trình bày đoạn code bên dưới

Reply:
em viết hàm này để gọi ra hàm lấy bài viết theo tháng trong controller
em tạo mảng ```dataChart``` để chứa mảng các bài viết trong thangs
```dataMonth``` sẽ lấy ra keys(tháng) của data sau đó em dùng map js để tạo ra mảng mới với các số tháng được làm tròn 
```valuePost``` lấy ra giá trị là số lượng bài viết đã được duyệt 
trong vòng for em lặp index cho 12 tháng vì index chạy từ 0 nên e kiểm tra
nếu tháng đi từ 1 thì đẩy giá trị và keys vào mảng ```dataChart```
Sau đó em sử dụng thư viện chartJs, thay đổi các trường label là các tháng , data sẽ là mảng ```dataChart``` và type sẽ là 'bar'